.github/workflows/aws-oidc-deploy-test.yaml

```diff
@@ -16,7 +16,6 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      USE_LOCAL_BACKEND: true
```

Comment:
Because we no-longer source the config file then read variables from the environment, I need to `echo 'export USE_LOCAL_BACKEND=true' >> civiform_config.sh` in this variable instead.